src/puppetlabs/puppetdb/http/catalogs.clj

```diff
@@ -53,17 +41,15 @@
                       (str (:url-prefix globals))))
 
     [node "edges" &]
-    (-> (comp (edges/edges-app version false) (partial http-q/restrict-query-to-node node))
+    (-> (edges/edges-app version false (partial http-q/restrict-query-to-node node))
```

Comment:
Yes, this is an example of where I think the old way is actually better; unless passing the middleware as a parameter really buys us something, I'd rather see one fewer parameter that needs to be passed through in multiple places.